frontend/src/metabase/embedding/embedding-hub/components/StepperWithCards/StepperWithCards.tsx

```diff
@@ -80,7 +89,7 @@ export const StepperWithCards = ({ steps }: { steps: StepperStep[] }) => {
             className={cx(S.step, isDone && S.stepDone)}
             icon={isDone ? <Icon name="check" /> : undefined}
             description={
-              <Group align="stretch">
+              <Grid spacing="md">
```

Comment:
I checked Mantine and it really [doesn't have `spacing`](https://mantine.dev/core/grid/?t=props), I suspect since this didn't work, you should be able to remove it.